ui/app/pages/swaps/swaps-gas-customization-modal/swaps-gas-customization-modal.component.js

```diff
@@ -0,0 +1,218 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import PageContainer from '../../../components/ui/page-container'
+import { Tabs, Tab } from '../../../components/ui/tabs'
+import { calcGasTotal } from '../../send/send.utils'
+import {
+  sumHexWEIsToRenderableFiat,
+} from '../../../helpers/utils/conversions.util'
+import AdvancedGasInputs from '../../../components/app/gas-customization/advanced-gas-inputs'
+import BasicTabContent from '../../../components/app/gas-customization/gas-modal-page-container/basic-tab-content'
+
+export default class GasModalPageContainer extends Component {
+  static contextTypes = {
+    t: PropTypes.func,
+    metricsEvent: PropTypes.func,
+    trackEvent: PropTypes.func,
+  }
+
+  static propTypes = {
+    insufficientBalance: PropTypes.bool,
+    gasPriceButtonGroupProps: PropTypes.object,
+    infoRowProps: PropTypes.shape({
+      originalTotalFiat: PropTypes.string,
+      originalTotalEth: PropTypes.string,
+      newTotalFiat: PropTypes.string,
+      newTotalEth: PropTypes.string,
+      sendAmount: PropTypes.string,
+      transactionFee: PropTypes.string,
+      extraInfoRow: PropTypes.shape({ label: PropTypes.string, value: PropTypes.string }),
+    }),
+    onSubmit: PropTypes.func,
+    cancelAndClose: PropTypes.func,
+    customPriceIsSafe: PropTypes.bool,
+    disableSave: PropTypes.bool,
```

Comment:
This prop is never passed into the container

Reply:
Added to the container return in bbc4e7aa5

We will also need to augment the setting of this property with considerations for minimumGasPrice in https://github.com/MetaMask/metamask-extension/pull/9600/files